openfl-tutorials/interactive_api/Tensorflow_MNIST/envoy_one_folder/mnist_shard_descriptor.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Mnist Shard Descriptor."""
+
+import logging
+
+import numpy as np
+from tensorflow import keras
```

Comment:
Please avoid using Deep Learning frameworks in Shard Descriptor implementation.
Almost any data preparation procedure may be implemented with numpy or any other lightweight libs.
We try to limit DL framework usage to model definition and utilising their computational backend, both done on the frontend (jupyter)